packages/aws-cdk/lib/init-templates/app/util/os.ts

```diff
@@ -78,7 +72,7 @@ function hasAnyChars(...chars: string[]): (x: string) => boolean {
  */
 function posixEscape(x: string) {
   // Turn ' -> '"'"'
-  x = x.replace("'", "'\"'\"'");
+  x = x.replace(/'/g, "'\"'\"'");
```

Comment:
Eek! Good find!

Reply:
This is actually in, like, a million places. I decided not to update it everywhere in this PR but I'm going to do another that fixes this on a wider scale.